substra/sdk/utils.py

```diff
@@ -36,6 +36,7 @@ def path_leaf(path):
 
 @contextlib.contextmanager
 def extract_files(data, file_attributes):
+    # FIXME should be move to the backend module
```

Comment:
Could you implement these 2 fixme's? They seem simple enough to be included in this PR.

Reply:
extract_files and extract_data_sample_files are used in schema.py, which is in substra/sdk so it should stay there, I can remove the FIXME

Reply:
That's good for me :)